src/components/Observations/ObsCard.js

```diff
@@ -31,7 +31,7 @@ const ObsCard = ( { item, handlePress }: Props ): Node => (
     </View>
     <View>
       <Text style={textStyles.text}>{item.identificationCount}</Text>
-      <Text style={textStyles.text}>{item.commentCount}</Text>
+      <Text style={textStyles.text} testID="ObsList.obsCard.commentCount">{item.commentCount}</Text>
```

Comment:
Question here: how much test coverage should we be aiming for? Is it enough to test that one piece of data like `commentCount` is rendering correctly, or is it better to test a few different things?

Reply:
It's subjective. If you test every single thing, you have a lot of confidence it's working the way you intended, but you also spent a lot of time writing tests instead of writing the code under test, and you probably made a really brittle test that will require lots of maintenance every time you change the code under test. Snapshots seem like a way to reduce the labor of writing tests, but they don't address the brittleness issue. I was sort of thinking they might be more appropriate later in the development process when components achieve some degree of stability.

Personally, I wanted to test things like "if I provide this data, are the cards I expect getting rendered, AND do the cards have some of the content I'm expecting?" and not "do the cards have ALL the content I'm expecting?" because I imagine the content of the cards is going to change a lot in these early stages of development.

Reply:
Rad, makes sense to me. The RN testing course I took suggested the same thing about snapshots -- we might want to start using them once we're closer to an MVP and know the code is more stable.